packages/@ember/-internals/extension-support/lib/data_adapter.js

```diff
@@ -1,8 +1,119 @@
 import { getOwner } from '@ember/-internals/owner';
-import { scheduleOnce } from '@ember/runloop';
-import { get, objectAt, addArrayObserver, removeArrayObserver } from '@ember/-internals/metal';
+import { backburner } from '@ember/runloop';
+import { get } from '@ember/-internals/metal';
 import { dasherize } from '@ember/string';
 import { Namespace, Object as EmberObject, A as emberA } from '@ember/-internals/runtime';
+import { consumeTag, createCache, getValue, tagFor, untrack } from '@glimmer/validator';
+
+function iterate(arr, fn) {
+  if (Symbol.iterator in arr) {
```

Comment:
Can't use `Symbol` here  (it isn't defined on IE11)

Reply:
Hmm, this code should only run in Chrome/Firefox shouldn't it, since it's only for inspector? I can guard it regardless, just in case.

Reply:
Inspector supports (or at least did) running outside of an extension from a bookmarklet. You could inspect in other browsers this way (I've personally used it for Safari at least).